include/ensmallen_bits/problems/schaffer_function_n1.hpp

```diff
@@ -72,6 +72,12 @@ class SchafferFunctionN1
     {
         return std::pow(coords[0], 2);
     }
+    arma::Col<size_t> Shuffle(size_t& populationSize)
+    {
+      return arma::shuffle(arma::linspace<arma::Col<size_t> >(0,
+            populationSize -1));
```

Comment:
```suggestion
          populationSize -1));
```

Should be tabbed twice (4 spaces) - https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping